exporter/otlp-http/opentelemetry-exporter-otlp-http.gemspec

```diff
@@ -29,6 +29,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'opentelemetry-common', '~> 0.19.6'
   spec.add_dependency 'opentelemetry-exporter-otlp-common'
   spec.add_dependency 'opentelemetry-sdk', '~> 1.2'
+  spec.add_dependency 'opentelemetry-semantic_conventions'
```

Comment:
I think maybe we need a version constraint here, yeah?

Reply:
I think the only real reason is to prevent someone with a very old installation of `opentelemetry-semantic_conventions` from satisfying this constraint, since such an old version wouldn't have the constants needed.